src/prescriptions/prescriptions.scss

```diff
@@ -46,14 +38,17 @@ title {
 }
 
 .tabsContainer {
-  background-color: $ui-02;
+  padding: 0 spacing.$spacing-05;
+}
 
-  :global(.cds--tabs__nav-item--selected) {
-    box-shadow: inset 0 2px 0 0 var(--brand-03) !important;
+.tab {
+  &:active,
+  &:focus {
+    outline: 2px solid var(--brand-03) !important;
   }
 
-  :global(.cds--tab--list) button {
-    max-width: 12rem !important;
+  &[aria-selected='true'] {
```

Comment:
Does the same thing as `:global(.cds--tabs__nav-item--selected) {`, just more concisely.